caffe-grid/src/main/python/com/yahoo/ml/caffe/CaffeOnSpark.py

```diff
@@ -51,5 +51,5 @@ def trainWithValidation(self,train_validation_source):
 
         :param DataSource: A list containing the source for training and validation data
         """
-        self.__dict__.get('cos').trainWithValidation(train_validation_source)
+        return self.__dict__.get('cos').trainWithValidation(train_validation_source)
```

Comment:
I thought that we will avoid array, and call out trainSrc and validationSrc explicitly.